lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -190,18 +190,18 @@ <h3 class="course-title">
                           default_share_text = _("I'm taking {course_name} online with @edxonline. Check it out!").format(course_name=course_overview.display_name_with_default)
                           share_text = urllib.quote_plus(share_settings.get('DASHBOARD_TWITTER_TEXT', default_share_text.encode('utf-8')))
                           twitter_url = 'https://twitter.com/intent/tweet?text=' + share_text + '%20' + urllib.quote_plus(twitter_share_url)
+                          share_msg = ("Share {course_name} on Twitter").format(course_name=course_overview.display_name_with_default)
```

Comment:
I think you should wrap the text `Share {course_name} on Twitter` with gettext function, `_()`